File: CEPA_2015/Splash.vb
Public Class Splash

    Private Sub Timer1_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Timer1.Tick

        If (ProgressBar1.Value < ProgressBar1.Maximum) Then
            ProgressBar1.Increment(5)

        Else
            Timer1.Stop()
            Me.Hide()
            Jogo.Show()
        End If
    End Sub


    Private Sub ProgressBar1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ProgressBar1.Click

    End Sub

    Private Sub Pic11_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Pic11.Click

    End Sub
End Class

File: BeautyCompany/SysBeautyCompany/Splash.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace SysBeautyCompany
{
    public partial class FRMSPLASH : Form
    {
        public FRMSPLASH()
        {
            InitializeComponent();
            timer1.Start();

        }

        private void timer1_Tick(object sender, EventArgs e)
        {
            if (progressBar1.Value < progressBar1.Maximum) {
                progressBar1.Increment(20);
                label1.Text = progressBar1.Value + "%";
            }
                else {
                    timer1.Stop();
                    FRMLOGIN FRMLogin = new FRMLOGIN();
                    FRMLogin.Show();
                    //FRMPRIN FRMPrin = new FRMPRIN();
                    //FRMPrin.Show();
                    this.Hide();
                }
         }
     }
 }
